### PR TITLE
fix/cities only

### PR DESCRIPTION
**Pull request by matheusfillipe**
limit to cities only

### CODE REVIEW

**Comment by matheusfillipe** [comment]
there's no fix

**Comment by handyc** [comment]
hi

**Comment by matheusfillipe** [comment]
> hi

wassup

**Comment by Flu** [comment]
hello

**Comment by matheusfillipe** [comment]
> hello

wassup bro

File: src/apis.ts [hunk 1]
```diff
@@ -114,12 +114,13 @@ export async function batchFetchPopulateCityData(
 
 export async function geocodeCityName(cityInput: string): Promise<City | null> {
   try {
-    // Use Nominatim API for geocoding the city
+    // Use Nominatim API for geocoding the city, cities only
     const response = await axios.get(
       `https://nominatim.openstreetmap.org/search`,
       {
```
Comment (Flu) [inline src/apis.ts:120]:
```typescript
// Use Nominatim API for geocoding the city, cities only
    const response = await axios.get(
      `https://idontknowtypescript.org/search`,
      {
```

What is happening

Comment (matheusfillipe) [reply src/apis.ts:120]: Thanks for your feedback. Please open an issue and follow our [guidelines](https://neal.fun/password-game/)

Comment (Flu) [reply src/apis.ts:120]: i tried reading the guidelines, but its prompting me for a password and whatever i do, it says its incorrect

Comment (handyc) [reply src/apis.ts:120]: > i tried reading the guidelines, but its prompting me for a password and whatever i do, it says its incorrect

okay

Comment (matheusfillipe) [reply src/apis.ts:120]: > okay

okay

Comment (ValwareIRC) [reply src/apis.ts:120]: > > okay
> 
> okay

okay

Comment (handyc) [reply src/apis.ts:120]: hi

Comment (matheusfillipe) [reply src/apis.ts:120]: hello

Comment (handyc) [reply src/apis.ts:120]: hi

**Pull request closed without merging**